services/payments/tests/unit/test_services_payments_gateway.py

```diff
@@ -59,12 +59,10 @@ def external_secret_envs(project_tests_dir: Path) -> EnvVarsDict:
 def app_environment(
     monkeypatch: pytest.MonkeyPatch,
     app_environment: EnvVarsDict,
-    disable_rabbitmq_and_rpc_setup: Callable,
+    mock_patch_setup_rabbitmq_and_rpc: None,
```

Comment:
so I find this naming unclear. so this fixture disables rabbitmq right? why now call it something generic like "mock_patch_setup" instead of something like "with_disabled_rabbitmq" or "disabled_rabbitmq" ?

Reply:
- I like the `with_disabled_*` naming. It looses the information that we are mocking the setup function but i think it is reasonable
- I was thinking the other day about this dependency. It is a good example of a non-data dependency in a pipeline. You connect A->B but actually B does not receive any data from A, it just defines an execution order. There must be a name for type of relationship. The order of execution is also important. All those are good lessons for the osparc pipeline/workbench mechanism.